src/csharp/Grpc.Core/ChannelCredentials.cs

```diff
@@ -171,7 +204,20 @@ internal override bool IsComposable
 
         internal override ChannelCredentialsSafeHandle CreateNativeCredentials()
         {
-            return ChannelCredentialsSafeHandle.CreateSslCredentials(rootCertificates, keyCertificatePair);
+            return ChannelCredentialsSafeHandle.CreateSslCredentials(rootCertificates, keyCertificatePair, this.verifyPeerCallbackInternal);
+        }
+
+        private int VerifyPeerCallbackInternal(IntPtr host, IntPtr pem, IntPtr userData, bool isDestroy)
```

Comment:
nit: rename to VerifyPeerCallbackHandler to avoid confusion with the `verifyPeerCallbackInternal` member?

Reply:
Renamed